forte/processors/stave/stave_processor.py

```diff
@@ -0,0 +1,302 @@
+# Copyright 2019 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+'''
+Package Requirements:
+    tornado == 6.1
+    django == 3.2
+    forte
+    *stave (Required in future version)
+
+Required environment variables:
+*All of the variables below will be deprecated in future version.
+    FRONTEND_BUILD_PATH:
+        Absolute path (or relative path from PYTHONPATH)
+        to stave build folder. Example: "stave/build/"
+    ONTOLOGY_PATH:
+        Absolute path to the ontology file.
+    DJANGO_BACKEND_PATH:
+        Absolute path (or relative path from PYTHONPATH)
+        to django backend folder. Example: "stave/simple-backend/"
+'''
+
+import os
+import sys
+import json
+import time
+import logging
+import asyncio
+import threading
+import collections
+import webbrowser
+from typing import Dict, Any
+import requests
+
+import django
+from django.core.wsgi import get_wsgi_application
+
+from tornado.web import FallbackHandler, StaticFileHandler, \
+    RequestHandler, url, Application
+from tornado.httpserver import HTTPServer
+from tornado.ioloop import IOLoop
+from tornado.wsgi import WSGIContainer
+
+from forte.common import Resources
+from forte.common.configuration import Config
+from forte.data.data_pack import DataPack
+from forte.processors.base import PackProcessor
+from forte.common.exception import ProcessExecutionException
+
+logger = logging.getLogger(__name__)
+
+__all__ = [
+    "StaveProcessor"
+]
+
+
+class StaveProcessor(PackProcessor):
+
+    def __init__(self, *args, **kwargs):
```

Comment:
Do we need to pass in the wild card arguments? We normally make our processor using empty `init` but use our own `initialize` to control most behavior.